blueman/plugins/applet/TransferService.py

```diff
@@ -131,7 +132,8 @@ def on_action(action):
             on_action("accept")
 
     def _cancel(self):
-        self._notification.close()
+        if self._notification:
+            self._notification.close()
```

Comment:
I don't care about this one but I am wondering if it is even possible to have ``Cancel`` method being called before ``AuthorizePush``..